src/constants.ts

```diff
@@ -33,11 +33,9 @@ export const SONARCLOUD_PRODUCTION_URL = 'https://sonarcloud.io';
 
 export const SONARQUBE_JRE_PROVISIONING_MIN_VERSION = '10.5';
 
-export const SONAR_CACHE_DIR = path.join(
-  process.env.HOME ?? process.env.USERPROFILE ?? '',
-  '.sonar',
-  'cache',
-);
+export const SONAR_DIR = path.join(process.env.HOME ?? process.env.USERPROFILE ?? '', '.sonar');
```

Comment:
having logic here in the constants file is misleading (at least to me). Would it make sense to set these in the PROPERTIES and reference them from there?

Reply:
Agreed, to me this was part of [SCANNPM-20](https://sonarsource.atlassian.net/browse/SCANNPM-20) so didn't implement it there, I can work on this today after this PR 🤔